dev/NavigationView/NavigationView.cpp

```diff
@@ -154,6 +152,58 @@ NavigationView::NavigationView()
 
     Unloaded({ this, &NavigationView::OnUnloaded });
     Loaded({ this, &NavigationView::OnLoaded });
+
+    m_selectionModel.SingleSelect(true);
+    m_selectionChangedEventToken = m_selectionModel.SelectionChanged(winrt::auto_revoke, { this, &NavigationView::OnSelectionModelSelectionChanged });
+
+    m_navigationViewItemsFactory = winrt::make_self<NavigationViewItemsFactory>();
```

Comment:
>m_navigationViewItemsFactory = winrt::make_self<NavigationViewItemsFactory>(); [](start = 4, length = 78)

Why does this have to be done here instead of in the header? right now you have m_navigationViewItemsFactory{nullptr}; replacing with m_navigationViewItemsFactory{} should do what this line does shouldn't it?

Reply:
Yeah, as you saw, there is an issue with the object lifetime when doing this in the header. Haven't been able to determine why the change you proposed is problematic, so for now keeping the instantiation of this here.